docker-compose.yml

```diff
@@ -20,6 +20,8 @@ services:
       - ${NGINX_HTTP_EDITOR_PORT:-40080}:80
       - ${NGINX_HTTP_SUBSCRIBER_PORT:-40081}:81
       - ${NGINX_HTTPS_PORT:-40443}:443
+    extra_hosts:
+      - host.docker.internal:host-gateway
```

Comment:
I read somewhere this can resolve the `host.docker.internal` issue.  I don't know if it does though.